howto_docker.md

```diff
@@ -0,0 +1,58 @@
+# How to use Docker and Openfisca
```

Comment:
-> _How to use OpenFisca with Docker_ (to be consistent with [recipes.md](recipes.md))